### PR TITLE
BZ 1229990: expose stats port based on config

### PR DESCRIPTION
**Pull request by pweil-**
(was just exposing 1936)

### CODE REVIEW

File: pkg/cmd/admin/router/router.go [hunk 1]
```diff
@@ -156,6 +156,13 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 		glog.Fatal(err)
 	}
 
+	ports = append(ports, kapi.ContainerPort{
```
Comment (liggitt) [inline pkg/cmd/admin/router/router.go]: only expose the port if we're going to set up stats (non-zero port?)

Comment (pweil-) [inline pkg/cmd/admin/router/router.go]: updated.  The template already looks for > 0 so this works nicely to turn off stats completely.

**Comment by liggitt** [comment]
LGTM

**Comment by liggitt** [comment]
@smarterclayton merge?

**Comment by smarterclayton** [comment]
Approved [merge]

**Pull request merged**